rust/openvasd/src/controller/entry.rs

```diff
@@ -134,165 +107,128 @@ where
             }
         }
     }
+    tracing::debug!("{} {}", req.method(), kp);
 
     match (req.method(), kp) {
-        (&Method::HEAD, _) => Ok(ctx.response.empty(hyper::StatusCode::OK)),
         (&Method::POST, Scans(None)) => {
             match crate::request::json_request::<models::Scan>(&ctx.response, req).await {
                 Ok(mut scan) => {
-                    response_blocking(move || {
+                    if scan.scan_id.is_none() {
```

Comment:
Why is this `if` statement introduced again? In a previous PR we removed and even changed the API to not accept an scan_id for creating a scan

Reply:
rebase error, going to remove it.